libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2277,7 +2277,7 @@ bool AP_AHRS::attitudes_consistent(char *failure_msg, const uint8_t failure_msg_
             // check roll and pitch difference
             const float rp_diff_rad = primary_quat.roll_pitch_difference(ekf2_quat);
             if (rp_diff_rad > ATTITUDE_CHECK_THRESH_ROLL_PITCH_RAD) {
-                hal.util->snprintf(failure_msg, failure_msg_len, "EKF2 Roll/Pitch inconsistent by %d deg", (int)degrees(rp_diff_rad));
+                hal.util->snprintf(failure_msg, failure_msg_len, "EKF2: R/P inconsistent. Wait or check sensors");
```

Comment:
Also - does a pilot/user really know or care that it is EKF2 (or 3 or DCM) - answer no and it only confuses them.

Reply:
How about: "EKF2: Roll/Pitch difference. Wait or check sensors"
using "difference" instead of "inconsistent" saves 2 characters and gets it into 50 characters".

Reply:
its not a roll/pitch difference.....inconsistent is more accurate....alternatively... Atittude inconsistent

Reply:
If you used "EKF3: Attitude off by 10 deg. Wait/check sensors" you could also use the same message for Yaw (which is also attitude) - so you could save about 45 bytes of flash by reusing the string.

Actually if you used "%s: Attitude off by %d deg. Wait/check sensors" and pass "EKF3"/"EKF2"/"DCM" as a parameter you save even more.